js/views/itemShortVw.js

```diff
@@ -13,14 +13,16 @@ module.exports = Backbone.View.extend({
     'click .js-itemShortEdit': 'editItemClick',
     'click .js-itemShortDelete': 'deleteItemClick',
     'click .js-userShortFollow': 'followUser',
-    'click .js-userShortUnfollow': 'unfollowUser'
+    'click .js-userShortUnfollow': 'unfollowUser',
+    'click .js-blockUser': 'blockUser',
+    'click .js-unblockUser': 'unblockUser'
   },
 
   initialize: function(options){
     //pre-load image
     var self=this;
     this.parentEl = $(options.parentEl);
-    this.listenTo(this.model, 'change:priceSet', this.render);
+    this.listenTo(this.model, 'change', this.render);
```

Comment:
Double check to make sure this isn't causing extra renders.

The itemShort model has an update function that sets priceSet, which triggers a change event, which could cause a double render every time an itemShort view is created.

The update function should be moved into a parse function to fix that.